ambry-admin/src/main/java/com.github.ambry.admin/AdminSecurityService.java

```diff
@@ -187,7 +187,17 @@ private void setGetBlobResponseHeaders(RestResponseChannel restResponseChannel,
         restResponseChannel.setHeader("Content-Disposition", "attachment");
       }
     }
-    if (blobProperties.isPrivate()) {
+    setCacheHeaders(blobProperties.isPrivate(), restResponseChannel);
+  }
+
+  /**
+   * Sets headers that provide directions to proxies and caches.
+   * @param isPrivate whether the blob is private.
+   * @param restResponseChannel the channel that the response will be sent over
+   * @throws RestServiceException if there is any problem setting the headers
+   */
+  private void setCacheHeaders(boolean isPrivate, RestResponseChannel restResponseChannel) throws RestServiceException {
+    if (isPrivate) {
       restResponseChannel.setHeader(RestUtils.Headers.EXPIRES, new Date(0));
```

Comment:
Private blobs should not be cached and expire immediately.

While this method to indicate that is acceptable, the [recommended](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.21) method seems to be set it to be equal to the `Date` header. Will make the change.